diff --git a/vendor/md4c/src/md4c.c b/vendor/md4c/src/md4c.c
index 9905825..85b9935 100644
--- a/vendor/md4c/src/md4c.c
+++ b/vendor/md4c/src/md4c.c
@@ -6802,12 +6802,6 @@ md_analyze_line(MD_CTX* ctx, OFF beg, OFF* p_end,
             }
         }
 
-        /* Check whether we are table continuation. */
-        if(pivot_line->type == MD_LINE_TABLE  &&  n_parents == ctx->n_containers) {
-            line->type = MD_LINE_TABLE;
-            break;
-        }
-
         /* Check for ATX header. */
         if(line->indent < ctx->code_indent_offset  &&
                 off < ctx->size  &&  CH(off) == _T('#'))
@@ -6856,6 +6850,12 @@ md_analyze_line(MD_CTX* ctx, OFF beg, OFF* p_end,
             }
         }
 
+        /* Check whether we are table continuation. */
+        if(pivot_line->type == MD_LINE_TABLE  &&  n_parents == ctx->n_containers) {
+            line->type = MD_LINE_TABLE;
+            break;
+        }
+
         /* Check for table underline. */
         if((ctx->parser.flags & MD_FLAG_TABLES)  &&  pivot_line->type == MD_LINE_TEXT
             &&  off < ctx->size  &&  ISANYOF3(off, _T('|'), _T('-'), _T(':'))
